aep/general/0005/aep.yaml

```diff
@@ -0,0 +1,7 @@
+---
+id: 5
+state: approved
+slug: intro
+created: 2024-01-10
+placement:
+  category: general
```

Comment:
```suggestion
  category: meta
```

WDYT?

Reply:
meta IMO has more to do with the AEP governance and structure. For example, glossary is general, not meta. 

I'd like to keep it as general but open for discussion. perhaps in the weekly?